pyaedt/edb_core/nets.py

```diff
@@ -994,3 +994,55 @@ def find_and_fix_disjoint_nets(self, net_list=None, keep_only_main_net=False, cl
         self._logger.info_timer("Disjoint Cleanup Completed.")
         self._logger.info("Found {} objects in {} new nets.".format(len(disjoints_objects), len(new_nets)))
         return new_nets
+
+    @pyaedt_function_handler()
+    def convert_path_to_polygon(self, net_list=None):
```

Comment:
is this the use case? maybe customer just want to convert 1 trace to polygon. In that caseI would probably put it as a method of EDBPrimitive class. What do you think?